src/presentation/controller/transition/LimitedPartnershipController.ts

```diff
@@ -169,6 +179,58 @@ class LimitedPartnershipController extends AbstractController {
       }
     };
   }
+
+  sendPageData() {
+    return async (request: Request, response: Response, next: NextFunction) => {
+      try {
+        const { tokens, ids } = super.extract(request);
+        const pageType = super.extractPageTypeOrThrowError(request, TransitionPageType);
+        const pageRouting = super.getRouting(transitionRouting, pageType, request);
+
+        const result = await this.limitedPartnershipService.sendPageData(
+          tokens,
+          ids.transactionId,
+          ids.submissionId,
+          pageType,
+          request.body
+        );
+
+        if (result?.errors) {
+          response.render(
```

Comment:
the limitedPartnership is passed into the template during the getRouting() function to load the page, does it need to be passed in again when rendering the page with an error

Reply:
i'm thinking that if that code in teh controller gets executed:
`if (result?.errors) {
          response.render(`
then the template will render without everything it needs.

My view is that if we are keeping 
`if (result?.errors) {
          response.render(`
then we should provide what the template needs in the render.